Rock.Messaging.SQS/MQ/SQSMessagingScenarioFactory.cs

```diff
@@ -1,26 +1,45 @@
 ﻿using Amazon.SQS;
 using System;
+using System.Collections.Generic;
+using System.Linq;
 
+#if ROCKLIB
+namespace RockLib.Messaging.SQS
+#else
 namespace Rock.Messaging.SQS
+#endif
 {
     public class SQSMessagingScenarioFactory : IMessagingScenarioFactory
     {
-        private readonly ISQSConfigurationProvider _configurationProvider;
+        private ISQSConfigurationProvider _configurationProvider;
 
-        public SQSMessagingScenarioFactory(ISQSConfigurationProvider configurationProvider)
+#if ROCKLIB
```

Comment:
For `#if ROCKLIB` here, I think it needs both the default constructor (which is implicitly defined here), and the existing non-default `SQSMessagingScenarioFactory(ISQSConfigurationProvider configurationProvider)` constructor. I'm pretty sure that's how we're doing it elsewhere. Let's talk through this tomorrow.